app/tasks/usage.php

```diff
@@ -379,19 +396,19 @@
                     foreach ($projects as $project) {
                         $projectId = $project->getId();
 
-                        // Get total storage
+                        // storage.tags.total
```

Comment:
What does `tags` stand for? 🤔

Reply:
Those are the storage used by Function tags, previously it was added directly to sum of storage used by all files, now that we separate files into buckets, this seemed complex doing here, so I added different stat, so that we can combine in the front-end if we want.

Reply:
ah gotcha 👍🏻